default/python/turn_events/turn_events.py

```diff
@@ -31,4 +32,35 @@ def execute_turn_events():
         if fo.create_field(field_type, x, y, size) == fo.invalid_object():
             print >> sys.stderr, "Turn events: couldn't create new field"
 
+    gsd = fo.get_galaxy_setup_data()
+    # creating monsters
+    monster_freq = fo.monster_frequency(gsd.monsterFrequency)
+    # monster freq ranges from 30 (= one monster per 30 systems) to 3 (= one monster per 3 systems)
+    # (example: low monsters and 150 Systems results in 150 / 30 * 0.001 = 0.005)
+    if monster_freq > 0 and random() < len(systems) / monster_freq * 0.001:
+        if random() < 0.7:
+            monster_type = "SM_KRILL_1"
+        else:
+            monster_type = "SM_FLOATER"
+
+        print "...creating new", monster_type
+
+        # search for systems without planets or fleets
+        candidates = [s for s in systems if len(fo.sys_get_planets(s)) <= 0 and len(fo.sys_get_fleets(s)) <= 0]
+        system = choice(candidates)
+        # if system selection fails, report an error
+        if system == fo.invalid_object():
+            util.report_error("Python execute_turn_events: unable to find system for monster spawn")
```

Comment:
The `util.report_error` function originally has been designed for use during universe generation, it probably makes no sense using it in the turn events scripts. I'm not even sure if that might create issues.

For debug/error logging, I think it's best if you stick to the examples set in the already existing code for field spawning (that also applies to the text: instead of prefixing with "Python execute_turn_events:" use the simpler and shorter "Turn events:").